setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(
+    name='libero schemas',
+    version='0.1',
```

Comment:
```suggestion
    version='0.0.1',
```

(Sounds a pain to have to include this in the code.)

Reply:
This can be read from git tags but this project doesn't have any yet.